bin/GameIndex.yaml

```diff
@@ -33582,7 +33582,7 @@ SLUS-20640:
   region: "NTSC-U"
   compat: 5
 SLUS-20641:
-  name: "IndyCar Series"
+  name: "IndyCar Series featuring the Indianapolis 500"
```

Comment:
As far as I can tell, it's just IndyCar Series

https://steemitimages.com/640x0/https://cdn.steemitimages.com/DQmQxhbr2QpeNy1FTAqXtKYzu6zRiQy474KzjqZuafRM77u/image.png